### ISSUE TITLE
Error on dapr init with podman/linux

### ISSUE THREAD

**Issue by matzew** [dapr/cli#1180]
## Expected Behavior

`dapr init` works,

## Actual Behavior

does not work, on my box (Fedora and podman)

```
docker (or podman) version
WARN[0000] Failed to decode the keys ["storage.options.ostree_repo"] from "/home/matzew/.config/containers/storage.conf" 
Client:       Podman Engine
Version:      4.3.1
API Version:  4.3.1
Go Version:   go1.19.2
Built:        Fri Nov 11 16:01:27 2022
OS/Arch:      linux/amd64
```

Here is the error:

```
apr init        
⌛  Making the jump to hyperspace...
ℹ️  Container images will be pulled from Docker Hub
ℹ️  Installing runtime version 1.9.5
❌  Downloading binaries and setting up components...
❌  time="2023-01-25T10:12:10+01:00" level=warning msg="Failed to decode the keys [\"storage.options.ostree_repo\"] from \"/home/matzew/.config/containers/storage.conf\""
Error: short-name resolution enforced but cannot prompt without a TTY
```

## Release Note

RELEASE NOTE:

**Comment by matzew** [comment]
yep, it is installed, and I am using it. Similar to the output above, with `--version` I am getting 

```
podman --version
podman version 4.3.1
```

**Comment by pravinpushkar** [comment]
It seems like it is already a known issue with Podman. Just searching for the above error got me many such occurrences. Not sure what is the right solution as I have not tried these things.

https://github.com/containers/podman/issues/11745
https://github.com/containers/podman/issues/11530
https://github.com/containers/podman/issues/11425

**Comment by matzew** [comment]
@pravinpushkar I built the `cli` on my box, and now it work.Perhaps a new releae of the cli would be appreciated?

**Comment by mukundansundar** [comment]
The dapr release is scheduled shortly this month. Please see the 1.10 planning issue in dapr/dapr repo.

**Comment by matzew** [comment]
@mukundansundar The actual fix is here: https://github.com/dapr/cli/pull/1186/files

I came to this conclusion after checking the related podman issues, and ending up here: https://github.com/kubernetes-sigs/kind/issues/2186

Checking the code told me that there is now `docker.io` used as the fully qualified name.

closing this ticket, as the local build works. Thanks for your help

/close

**Comment by mukundansundar** [comment]
> @mukundansundar The actual fix is here: https://github.com/dapr/cli/pull/1186/files
> 
> I came to this conclusion after checking the related podman issues, and ending up here: [kubernetes-sigs/kind#2186](https://github.com/kubernetes-sigs/kind/issues/2186)
> 
> Checking the code told me that there is now `docker.io` used as the fully qualified name.
> 
> closing this ticket, as the local build works. Thanks for your help
> 
> /close

closing issue.

**Comment by matzew** [comment]
Fixed by https://github.com/dapr/cli/pull/1186

**Comment by pravinpushkar** [comment]
@9034725985 Please note that the fix will be released with 1.10, which is due this week.
